test/read-only-handler.spec.ts

```diff
@@ -271,4 +271,41 @@ describe('ReadOnlyHandler', () => {
             doNothing(readOnly.foo);
         }).not.toThrow();
     });
+
+    describe.skip('issue#20 - getOwnPropertyDescriptor', () => {
+        it('should return reactive proxy when property accessed via raw getter', () => {
+            const target = new ReactiveMembrane();
+            const todos = {};
+            Object.defineProperty(todos, 'entry', {
+                get() {
+                    return { foo: 'bar' };
+                },
+                configurable: true
+            });
+
+            const proxy = target.getReadOnlyProxy(todos);
+            const desc = Object.getOwnPropertyDescriptor(proxy, 'entry');
+            const { get } = desc;
+            expect(() => { 
+                get().foo = '';
+            }).toThrow();
```

Comment:
perhaps this should become a test of its own?

Reply:
The second assertion(L292) is verifying that the data wasn't mutated. To separate it to its own test case I would have to copy the whole thing, place the mutation attempt(L290) in a try-catch. Seems too much duplication for asserting that read only proxy is blocking the mutation.

And I have the test case description wrong, will fix that.

Reply:
Both the tests exploit the same loop hole in the implementation.